web/src/components/netflow-table.tsx

```diff
@@ -22,10 +22,11 @@ import { Column } from '../utils/columns';
 const NetflowTable: React.FC<{
   flows: ParsedStream[];
   columns: Column[];
+  size: string;
```

Comment:
there could also be a union type here (`'s' | 'm' | 'l'`) to enforce stronger typing, like you did for `width` below?
this pattern is widely used in patternfly too (e.g. https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Text/Text.tsx#L22) I guess it's a good practice to adopt too